emodpy_typhoid/interventions/typhoid_vaccine.py

```diff
@@ -25,7 +25,7 @@ def new_intervention( camp, efficacy=0.82, mode="Shedding", constant_period=0, d
          constant_period (float, optional): The constant period of the waning effect in days. Default is 0.
          decay_constant (float, optional): The decay time constant for the waning effect. Default is 6935.0.
          expected_expiration (float, optional): The mean duration before efficacy becomes 0. If this is set to non-zero value, the constant_period and decay_constant are ignored. These are two different modes of waning.
-
+         deduplication_policy (string, optional): Defaults to 'replace'. Can also be 'combine'. 
```

Comment:
line 28 for docstring should not be here. Should be moved to def new_vax function's doc string